user/customizing-the-build.md

```diff
@@ -264,10 +264,10 @@ branches and tags that start with `deploy-` in any combination of cases.
 ## Skipping a build
 
 If you don't want to run a build for a particular commit, because all you are
-changing is the README for example, add `[ci skip]` to the git commit message.
+changing is the README for example, add `[ci skip]` or `[skip ci]` to the git commit message.
 
-Commits that have `[ci skip]` anywhere in the commit messages are ignored by
-Travis CI.
+Commits that have `[ci skip]` or `[ci skip]` anywhere in the commit messages are ignored by
```

Comment:
I'd love to just tell people about one of these not two. It does not serve any useful purpose.